src/slim-select/select.ts

```diff
@@ -85,7 +85,7 @@ export default class Select {
   }
 
   private observeCall(mutations: MutationRecord[]): void {
-    // If we are not listeing do nothing
```

Comment:
I like to describe things in the order of the functionality.

Describe what the if statement is checking, then describe what will happen in the if statement

Reply:
This was actually because of my IDE pestering me about the sentence structure. I'll revert it

Reply:
haha an ide talking about sentence structure